src/main/java/za/ac/cput/school_management_grp33/util/Util.java

```diff
@@ -1,9 +1,23 @@
 package za.ac.cput.school_management_grp33.util;
 
+import org.apache.commons.lang3.StringUtils;
+
 /**
  * Utility Class, also known as Helper class.
  * It contains a bunch of related methods,
  * so they can be reused across the application.
  */
 public final class Util {
+    public static boolean isEmptyOrNull(String str) {
+        return StringUtils.isEmpty(str);
```

Comment:
once you remove the apache-commons dependency, change this to `return StringUtils.hasText(str);`
Does the same thing